### DISCUSSION TITLE
Comments on commits not showing

### DISCUSSION THREAD

**Discussion by Jai-JAP** [community/community#61420] [Mobile]
### Bug

GitHub Mobile shows notifications for comments on commits but when opening it no comments can be seen. 
This feels very buggy. 

Please fix it by either hiding these type of notifications on GitHub mobile or by adding ability to see comments made on commits on mobile.

**Comment by aconsuegra** [comment]
Hi @Jai-JAP, thanks for reaching out with your feedback. You are correct, the Mobile apps currently don't support comments on commits, and we probably shouldn't display notifications about them to avoid confusion. I will report this issue back to our engineering team, hopefully we can work on a fix.

**Reply by jsoref** [reply]
Yes, if you aren't going to show the comment, you really shouldn't show the notification:
![Screenshot_20230811-113751.png](https://github.com/community/community/assets/2119212/f5b6be25-cca9-4edc-90a5-f8bbe5665fb4)

https://github.com/check-spelling/radis/commit/a4ab6957e60e48a877151c34408e50d7115b3f15#commitcomment-124259834

I'd hope you could filter these out quickly until you implement the ability to show the comment ... 

Note that even including a web link to the comment would be an improvement over the current behavior...

**Reply by d-kleine** [reply]
Issue still persists, new comments displayed in Notifications, but when clicking the the commit, comments won't be displayed. Any update on that?

As an idea, would it be possible to add an extra button, tab or symbol in the commit view that displays the comments in this commit on a new page?
![gh-mobile-commit-nocomments](https://github.com/community/community/assets/53251018/e9ffd71b-0163-4967-9f20-1c7c441d73ab)

**Comment by jsoref** [comment]
Fwiw, I'm assuming this is reporting about Android.
The iOS issue for this is: https://github.com/orgs/community/discussions/28069

**Reply by jsoref** [reply]
This might be a duplicate of a year old ticket: https://github.com/orgs/community/discussions/16320
